docs/specs/app2app.md

```diff
@@ -155,6 +155,7 @@ The following parameter will be added to constructor of `Authgear`:
   - `App2AppOptions` contains the following fields:
     - `isEnabled`: If true, new sessions will be prepared for participating in app2app authentication.
     - `isInsecureDeviceKeyBindingEnabled`: If true, the sdk will try to create and bind device key during `approveApp2AppAuthenticationRequest` to "migrate" to app2app sessions.
+    - `state`: Opaque value used to maintain state between the request and the callback. Same purpose as [state in oidc](https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest).
```

Comment:
state should be provided in App2AppAuthenticateOptions, not when Authgear is configured.

Reply:
Sorry I put it at a wrong place, fixed thanks!